torch/fx/experimental/symbolic_shapes.py

```diff
@@ -90,17 +138,22 @@ def __bool__(self):
 }
 
 for method, _func in magic_methods.items():
-    method_name = f'{method}'
-
     def _create_magic_impl(func):
+        method_name = method
+
         def magic_impl(self, other):
+            if SYM_FUNCTION_MODE:
+                return _handle_sym_dispatch(getattr(operator, method_name), (self, other), {})
             if isinstance(other, PySymInt):
                 other = other.expr
             return PySymInt(func(self.expr, other), self.shape_env)
```

Comment:
Do we want to potentially propagate constants here, now that we're storing constants directly on the `PySymInt`? (maybe this will change once we handle ops like nonzero)

Reply:
We could, though I don't think it's really that necessary as you shouldn't have called into SymInt in that case.